apps/testrunner/test_suites/selenium_tests/selenium_timer.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+class SeleniumTimer(object):
+
+    def __init__(self, driver):
+        self.driver = driver
+        self.measurements = []
+
+    def start(self):
+        self.start_timestamp = self.driver.execute_script("return new Date().getTime()")
+
+    def mark_measurement_point(self, url):
+        timing = self.driver.execute_script("return window.performance.timing")
+
+        self.measurements.append(dict(
```

Comment:
Yeah, but then you need to use apostrophes for key names, so this way is also widely used or am I missing something?